Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -59,6 +59,12 @@ const handledEvents = [
   'StartInteraction',
   'Interaction',
   'EndInteraction',
+  'StartXrSelect',
+  'XrSelect',
+  'EndXrSelect',
+  'StartXrSqueeze',
+  'XrSqueeze',
+  'EndXrSqueeze',
```

Comment:
I think capitalizing the 'R' in 'XR' for consistency is a good idea here.

Reply:
still relevant?